oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/TCPOALDefine.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.skywalking.oap.server.receiver.envoy;
+
+import org.apache.skywalking.oap.server.core.oal.rt.OALDefine;
+
+/**
+ * OAL rules to calculate TCP-specific metrics.
+ */
+public class TCPOALDefine extends OALDefine {
+    public static final TCPOALDefine INSTANCE = new TCPOALDefine();
+
+    private TCPOALDefine() {
+        super(
+            "oal/tcp.oal",
+            "org.apache.skywalking.oap.server.core.source",
+            "org.apache.skywalking.oap.server.core.source.tcp"
```

Comment:
Could we just keep 2 parameters here, and use `org.apache.skywalking.oap.server.core.source.tcp` directly?

Reply:
> Could we just keep 2 parameters here, and use `org.apache.skywalking.oap.server.core.source.tcp` directly?

`org.apache.skywalking.oap.server.core.source` is the source entity package name, `Service`, `ServiceInstance`, etc., they exist in package `org.apache.skywalking.oap.server.core.source`, we cannot set to others (`*.tcp`)

Reply:
Got it. Thanks.